CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+##TBD
+
+### Changed
+
+- (react-native): Update bugsnag-cocoa to v6.6.3
+  - Fix hang after receiving signals in some cases. [bugsnag-cocoa1002](https://github.com/bugsnag/bugsnag-cocoa/pull/1002)
+  - Improve performance of thread recording for handled errors. [bugsnag-cocoa#992](https://github.com/bugsnag/bugsnag-cocoa/pull/992)
```

Comment:
This update covers 6.6.2 and 6.6.3, so I think it can be consolidated as:

```suggestion
- (react-native): Update bugsnag-cocoa to v6.6.3
  - Improve performance of thread recording for handled errors [bugsnag-cocoa#992](https://github.com/bugsnag/bugsnag-cocoa/pull/992)
  - Stop Bugsnag unregistering other signal handlers when catching a mach exception [bugsnag-cocoa#976](https://github.com/bugsnag/bugsnag-cocoa/pull/976)/[bugsnag-cocoa#1002](https://github.com/bugsnag/bugsnag-cocoa/pull/1002)
  - Fix a tvOS file permission error that was introduced in 6.5.1 [bugsnag-cocoa#996](https://github.com/bugsnag/bugsnag-cocoa/pull/996)
  - Fix an analyzer warning [bugsnag-cocoa#994](https://github.com/bugsnag/bugsnag-cocoa/pull/994)

```